docs/references/uvh5_memo.tex

```diff
@@ -212,6 +206,93 @@ \subsection{Required Parameters}
   ECEF. The conversion between LLA and ECEF is given by WGS84. This is a
   two-dimensional array of size (Nants\_telescope,
   3). (\textit{antenna\_positions})
+\item \textbf{phase\_center\_catalog}: A series of nested datasets, similar to
+a dict in python (\textit{phase\_center\_catalog}). The top level keys are integers
+giving the phase center catalog IDs which are used to identify which
+baseline-times are phased to which phase center via the phase\_center\_id\_array.
```

Comment:
should `phase_center_id_array` be emphasized here to indicate it is an attribute on the object? It seems like other things are done that way? Or am I misinterpreting why emphasis is used?